### PR TITLE
drainer: Ignore all tidb placement operations

### LINKED ISSUE #1140: Support ignoring all placement operations for drainer
## Feature Request

**Describe the feature you'd like:**

See: https://github.com/pingcap/tidb/issues/32828

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

### PR DESCRIPTION
**Pull request by lcwangchao**
### What problem does this PR solve?

Issue Number: close #1140

### What is changed and how it works?

Set `tidb_placement_mode` to `ignore` in DSN to ignore all tidb placement operations.

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
Upstream and downstream are all TiDB. In update stream:

```
create placement policy pn followers=1;
create table t1(id int) placement policy pn;
```

The result in down stream:

```
mysql> show placement;
Empty set (0.01 sec)

mysql> show create table t1;
+-------+-------------------------------------------------------------------------------------------------------------+
| Table | Create Table                                                                                                |
+-------+-------------------------------------------------------------------------------------------------------------+
| t1    | CREATE TABLE `t1` (
  `id` int(11) DEFAULT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin |
+-------+-------------------------------------------------------------------------------------------------------------+
1 row in set (0.00 sec)
```

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to update the `tidb-ansible` repository
 - Need to be included in the release note

### Release note

- No release note

### CODE REVIEW

File: pkg/loader/util.go [hunk 1]
```diff
@@ -164,7 +167,8 @@ func createDBWitSessions(dsn string, params map[string]string) (db *gosql.DB, er
 	}
 
 	for k, v := range support {
-		dsn += fmt.Sprintf("&%s=%s", k, url.QueryEscape(v))
+		// For key='value', the value should be quoted to prevent it from being regarded as a key word
```
Comment (lance6716) [inline pkg/loader/util.go:170]: https://github.com/go-sql-driver/mysql#system-variables

you can see the example `time_zone=%27Europe%2FParis%27: SET time_zone='Europe/Paris'`

**Review by lance6716 — APPROVED** [review]

**Review by lichunzhu — APPROVED** [review]
LGTM

**Pull request merged**